sos/collector/transports/control_persist.py

```diff
@@ -193,6 +193,13 @@ def remote_exec(self):
                             f"{self.opts.ssh_user}@{self.address}")
         return self.ssh_cmd
 
+    def _copy_file_to_remote(self, fname, dest):
+        cmd = (f"/usr/bin/scp -oControlPath={self.control_path} "
+               f"{fname} {self.opts.ssh_user}@{self.address}:{dest}")
+        res = sos_get_command_output(cmd,
+                                     timeout=10)
```

Comment:
Nit: this can fit on one line

Reply:
I'll change it, don't recall why its split since it doesn't seem long enough for flake8 to have complained about it. I may have removed a different parameter.

This does bring up a question though, is 3.6 still the default python to target? I don't recall where, maybe looking at CI tests or some code in the actual sos files, but something last week looked to be using a minimum version of Python 3.8.

If were not minimum 3.8 yet, and it wasn't just in the CI, I will track down what it was an open an issue.

Reply:
> This does bring up a question though, is 3.6 still the default python to target? I don't recall where, maybe looking at CI tests or some code in the actual sos files, but something last week looked to be using a minimum version of Python 3.8.

Python3.6 is not further the minimal version, 3.8 it is. We already started to use e.g. walrus operator.